presto-docs/src/main/sphinx/release/release-331.rst

```diff
@@ -0,0 +1,83 @@
+=========================
+Release 331 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Prevent query failures when worker is shut down gracefully. (:issue:`2648`)
+* Fix join failures for queries involving OR predicate with non comparable functions. (:issue:`2861`)
+* Ensure query completed event is fired when there is an error during analysis or planning. (:issue:`2842`)
+* Fix query memory accounting for ``ORDER BY`` queries. (:issue:`2612`)
+* Fix :func:`last_day_of_month` for ``timestamp with time zone`` values. (:issue:`2851`)
+* Fix excessive runtime when parsing deeply nested expressions with unmatched parenthesis. (:issue:`2968`)
+* Correctly reject ``date`` literals that cannot be represented in Presto. (:issue:`2888`)
+* Improve query performance by removing redundant data reshuffling. (:issue:`2853`)
+* Improve performance of inequality joins involving ``BETWEEN``. (:issue:`2859`)
+* Improve engine performance in joins. (:issue:`2862`)
+* Enable dynamic filtering by default. (:issue:`2793`)
+* Show reorder join cost in ``EXPLAIN ANALYZE VERBOSE`` (:issue:`2725`)
+* Allow configuring resource groups selection based on user's groups. (:issue:`3023`)
+
+Security Changes
+----------------
+
+* Remove unused ``internal-communication.jwt.enabled`` configuration property. (:issue:`2709`)
+* Rename JWT configuration properties from ``http.authentication.jwt.*`` to ``http-server.authentication.jwt``. (:issue:`2712`)
+* Add access control checks for query execution, view query, and kill query. This can be configured using
+  :ref:`query_rules` in the file based system access control. (:issue:`2213`)
+* Prevent displaying columns of tables that user has no privilege granted in file based access control. (:issue:`2925`)
+* Add ``ALTER SCHEMA ... SET AUTHORIZATION`` syntax that allows to change the owner of the schema. (:issue:`2673`)
+
+Web UI Changes
+--------------
+
+* Fix copying worker address to clipboard. (:issue:`2865`)
+* Fix copying of query ID to clipboard. (:issue:`2872`)
+* Fix display of data size values. (:issue:`2810`)
+* Fix redirect from ``/`` to ``/ui/`` when Presto is run behind a proxy. (:issue:`2908`)
+* Fix ``Prepared Query`` section for :doc:`/sql/prepare` queries. (:issue:`2784`)
+* Display physical input read rate in UI. (:issue:`2873`)
+* Add simple form based authentication. Users can be authenticated with either the LDAP or the file based
+  authentication services. (:issue:`2755`)
+* Add support for disabling the web ui.  The UI can be disabled with the ``web-ui.enabled`` configuration
```

Comment:
```
* Allow disabling the UI using the ``web-ui.enabled`` configuration property. (:issue:`2755`)

Reply:
I made a minor change to your suggested text. Instead of "using the ...", I replaced it with "via the ..."